### DISCUSSION TITLE
Is there a way to `ssh` into a Vercel deployment to see which commands do and don't exist?

### DISCUSSION THREAD

**Discussion by hirehamir** [vercel/vercel#9918] [Help]
### Problem Description

It'd be useful if users had a way to `ssh` into deployments to see, for example, if `ffmpeg` or other commands work on the remote machine. I only saw 2 search results for `ssh` at https://vercel.com/docs, and neither looked relevant.

![image](https://user-images.githubusercontent.com/43425812/236645529-19461e80-0074-45fe-96fa-fe295496f97a.png)

At a high level, what I'm trying to do is convert an `.m4a` file to an `.mp3` file, and the libraries below work just fine locally, but not on my Vercel deployment, and I'm trying to see if `ffmpeg` is installed on the remote machine to explore that as a possible avenue for troubleshooting.

```ts
const ffmpegPath = require('@ffmpeg-installer/ffmpeg').path;
const ffmpeg = require('fluent-ffmpeg');
```

**Comment by xts-bit** [comment]
@hamirmahal Did you find any solution?

**Reply by hirehamir** [reply]
@xts-bit I didn't find a way to do this.
